website/docs/cli.md

```diff
@@ -283,9 +283,9 @@ Description:               Run Compaction for given instant time
 
  Keyword:                  compactionInstant
    Help:                   Base path for the target hoodie table
-   Mandatory:              true
+   Mandatory:              false
    Default if specified:   '__NULL__'
-   Default if unspecified: '__NULL__'
+   Default if unspecified: Earliest instant on the Hudi timeline
```

Comment:
One thing to clarify, for `compaction run` command in Hudi CLI, the `compactionInstant` argument is still needed.  I haven't changed the logic of Hudi CLI in the corresponding PR.

Reply:
Good catch thanks, I updated